### PR TITLE
#1719 Pick Sandbox is now shown at header when user is a curator

### PR DESCRIPTION
**Pull request by rabingaire**
**What kind of change does this PR introduce?**
Added feature #1719 where Pick Sandbox is added at header when the user is the curator.

**What is the current behavior?**
#1719 

**What is the new behavior?**
The user who is a curator will see the Pick button at the header of the sandbox.

**Checklist**:
- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [ ] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx [hunk 1]
```diff
@@ -217,12 +240,16 @@ const Header = ({ store, signals, zenMode }: Props) => {
...
           />
         )}
-        <ShareButton
-          style={{ fontSize: '.75rem', margin: '0 1rem' }}
-          signals={signals}
-          secondary={!sandbox.owned}
-          store={store}
-        />
+
+        {store.user.curatorAt && (
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx]: It think there needds to be a nullcheck here to be sure.

```suggestion
        {store.user && store.user.curatorAt && (
```

File: packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx [hunk 2]
```diff
@@ -231,6 +258,13 @@ const Header = ({ store, signals, zenMode }: Props) => {
           store={store}
         />
 
+        <ShareButton
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx]: Ahh, I've probably been confusing with the design! We should have this button before the "Fork" button. It changed later in the designs 😄

File: packages/app/src/app/store/modules/editor/index.js [hunk 3]
```diff
@@ -82,6 +83,7 @@ export default Module({
     moduleDoubleClicked: sequences.unsetDirtyTab,
     tabClosed: sequences.closeTab,
     tabMoved: sequences.moveTab,
+    pickSandboxModal,
```
Comment (CompuIves) [inline packages/app/src/app/store/modules/editor/index.js]: I believe this one is already under `signals.explore.pickSandboxModal`, so you could remove this change and run `signals.explore.pickSandboxModal(...)` from the header.

File: packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx (hunk 1 above)
Comment (rabingaire) [reply packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx]: 👍

File: packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx (hunk 2 above)
Comment (rabingaire) [reply packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx]: Yeah I thought you wanted to move the position of the button 👍 anyway now buttons are there as they were before.

File: packages/app/src/app/store/modules/editor/index.js (hunk 3 above)
Comment (rabingaire) [reply packages/app/src/app/store/modules/editor/index.js]: 👍

Comment (rabingaire) [reply packages/app/src/app/store/modules/editor/index.js]: Lack on knowledge on Mobx wrote that code haha 👍

**Comment by rabingaire** [comment]
Made changes as requested.

**Comment by SaraVieira** [comment]
@all-contributors please add @rabingaire for code

**Comment by SaraVieira** [comment]
Thank you so much! This looks great :D

**Pull request merged**